src/server/scripts/Kalimdor/zone_durotar.cpp

```diff
@@ -177,9 +178,571 @@ struct npc_mithaka : ScriptedAI
     bool _shipInPort;
 };
 
+// Echo Isles
+
+class spell_summon_zuni : public SpellScript
```

Comment:
any class belonging to a given zone_name must have such name in its name, e.g.: spell_durotar_summon_zuni

Reply:
fixed for next commit.